pyart/aux_io/rainbow_wrl.py

```diff
@@ -0,0 +1,493 @@
+    'Vvu': 'unfiltered_velocity_vv', # non standard name
+    'dBZ': 'reflectivity',
+    'dBZv': 'reflectivity_vv',      # non standard name
+    'dBuZ': 'unfiltered_reflectivity', # non standard name
+    'dBuZv': 'unfiltered_reflectivity_vv', # non standard name
+    'ZDR': 'differential_reflectivity',
+    'ZDRu': 'unfiltered_differential_reflectivity', # non standard name
+    'RhoHV': 'cross_correlation_ratio',
+    'RhoHVu': 'unfiltered_cross_correlation_ratio', # non standard name
+    'PhiDP': 'differential_phase',
+    'uPhiDP': 'uncorrected_differential_phase', # non standard name
+    'uPhiDPu': 'uncorrected_unfiltered_differential_phase', # non standard name 
+    'KDP': 'specific_differential_phase',
+    'uKDP': 'uncorrected_specific_differential_phase', # non standard name
+    'uKDPu': 'uncorrected_unfiltered_specific_differential_phase', # non standard name
+    'SQI': 'signal_quality_index', # non standard name
+    'SQIv': 'signal_quality_index_vv', # non standard name
+    'SQIu': 'unfiltered_signal_quality_index', # non standard name
+    'SQIvu': 'unfiltered_signal_quality_index_vv', # non standard name
+    'TEMP': 'temperature', # non standard name
+    'ISO0': 'iso0', # non standard name
+}
+
+
+def read_rainbow_wrl(filename, field_names=None, additional_metadata=None, file_field_names=False, exclude_fields=None, **kwargs):
+    """
+    Read a RAINBOW file.
+    
+    Parameters
+    ----------
+    filename : str
+        Name of the RAINBOW file to read.
+    field_names : dict, optional
+        Dictionary mapping RAINBOW field names to radar field names. If a
+        data type found in the file does not appear in this dictionary or has
+        a value of None it will not be placed in the radar.fields dictionary.
+        A value of None, the default, will use the mapping defined in the
+        Py-ART configuration file.
+    additional_metadata : dict of dicts, optional
+        Dictionary of dictionaries to retrieve metadata during this read.
+        This metadata is not used during any successive file reads unless
+        explicitly included.  A value of None, the default, will not
+        introduct any addition metadata and the file specific or default
+        metadata as specified by the Py-ART configuration file will be used.
+    file_field_names : bool, optional
+        True to use the MDV data type names for the field names. If this
+        case the field_names parameter is ignored. The field dictionary will
+        likely only have a 'data' key, unless the fields are defined in
+        `additional_metadata`.
+    exclude_fields : list or None, optional
+        List of fields to exclude from the radar object. This is applied
+        after the `file_field_names` and `field_names` parameters.
+    
+    
+    Returns
+    -------
+    radar : Radar
+        Radar object containing data from RAINBOW file.
+    
+    """
+    
+    # check if it is the right file. Open it and read it
```

Comment:
`_test_arguments(kwargs)` should likely be called here to check for non empty keyword arguments.